### ISSUE TITLE
rpm installation package disappears

### ISSUE THREAD

**Issue by init232** [greenplum-db/gpdb#17248]
Why is there no version of the rpm installation package in tags?

**Comment by xentinel3** [comment]
I was thinking the same, I even downloaded the 7.0.0 rpm package on december and now it has been deleted, that makes even harder to use greenplum because I think that now we have to compile and then install it

**Comment by tonywasson** [comment]
I don't know why, but RPMs were removed in March according to Wayback Machine on archive.org. 

March 8, 2024 --  "download now" button on greenplum.org was pointing at GitHub file releases page https://network.pivotal.io/products/vmware-greenplum

March 24, 2024 --  "download now" button started pointing to https://network.pivotal.io/products/vmware-greenplum. 

After logging in and accepting the EULA, RPMs are available. I checked v6.11.1 RPMs and v.7.10 RPMs. I've attached the EULAs accepted since these look like the normal commercial releases of Greenplum. For instance, the 7.10 version allows a 30 day evaluation. 

[greenplum.org-6.11.1.pdf](https://github.com/greenplum-db/gpdb/files/14871789/greenplum.org-6.11.1.pdf)
[greenplum.org-7.10.pdf](https://github.com/greenplum-db/gpdb/files/14871790/greenplum.org-7.10.pdf)

**Comment by xentinel3** [comment]
> I don't know why, but RPMs were removed in March according to Wayback Machine on archive.org.
> 
> March 8, 2024 -- "download now" button on greenplum.org was pointing at GitHub file releases page https://network.pivotal.io/products/vmware-greenplum
> 
> March 24, 2024 -- "download now" button started pointing to https://network.pivotal.io/products/vmware-greenplum.
> 
> After logging in and accepting the EULA, RPMs are available. I checked v6.11.1 RPMs and v.7.10 RPMs. I've attached the EULAs accepted since these look like the normal commercial releases of Greenplum. For instance, the 7.10 version allows a 30 day evaluation.
> 
> [greenplum.org-6.11.1.pdf](https://github.com/greenplum-db/gpdb/files/14871789/greenplum.org-6.11.1.pdf) [greenplum.org-7.10.pdf](https://github.com/greenplum-db/gpdb/files/14871790/greenplum.org-7.10.pdf)

This is the download page for the commercial version of greenplum, we're missing the rpms for Open source Greenplum

**Comment by ValeriiKretinin** [comment]
Any answers from devs? I can't download rpm packages even from https://network.pivotal.io/, because of

`As part of VMware’s integration with Broadcom starting May 6, 2024, new user sign-ups for Tanzu Network will be disabled. Tanzu Network and its associated registry will remain accessible to customers until at least May 31, 2024. Please stay tuned, soon there will be additional information regarding the transition.`

Just bring rpm packages back, guys

**Comment by ValeriiKretinin** [comment]
@gpopt

**Comment by init232** [comment]
I solved it through source code compilation, but there are many dependencies involved in the compilation process.

**Comment by ValeriiKretinin** [comment]
Yes, I am also considering this option to build my own RPM packages locally, but I checked CI and all the scripts that participate in the build, there are really too many dependencies, it all looks like crutches. It's sad that the developers are putting a spoke in the wheels

**Comment by ValeriiKretinin** [comment]
@init232 it looks like python2 is required to compile project, do you use python2 or something else or I'm doing smth wrong?

**Comment by init232** [comment]
> @init232 it looks like python2 is required to compile project, do you use python2 or something else or I'm doing smth wrong?

Yes. Requires source code to compile python2 and use soft link as default python
